sound/soc/sof/ipc4-topology.h

```diff
@@ -30,6 +30,8 @@
 #define SOF_IPC4_NODE_INDEX(x)	((x) & 0xff)
 #define SOF_IPC4_NODE_TYPE(x)  ((x) << 8)
 
+#define SOF_IPC4_INTEL_SSP_NODE_INDEX(x)	(((x) & 0xf) << 4)
```

Comment:
What is the difference between `SOF_IPC4_NODE_INDEX()` and `SOF_IPC4_INTEL_SSP_NODE_INDEX()` Does different type of DAIs have their node_index in different place?

Reply:
yes, made it a bit more intuitive. This is specific to SSP. DMIC will need a different encoding but that'll be a separate PR